app/presenters/details_presenter.rb

```diff
@@ -0,0 +1,41 @@
+require "govspeak"
+
+module Presenters
+  class DetailsPresenter
+    attr_reader :content_item_details, :body
+
+    def initialize(content_item_details)
+      @content_item_details = SymbolizeJSON.symbolize(content_item_details)
+      @body = content_item_details[:body]
+      @body = [@body] if @body.is_a?(Hash)
```

Comment:
No I didn't know about it either, probably because it is a rails method. good to know 👍